pkgs/os-specific/darwin/binutils/default.nix

```diff
@@ -37,10 +38,16 @@ stdenv.mkDerivation {
       ln -sf "${cctools}/bin/$i" "$out/bin/$i"
     done
 
-    # FIXME: this will give us incorrect man pages for bits of cctools
-    ln -s ${binutils-raw.out}/share $out/share
+    ln -s ${binutils-raw.bintools.out}/share $out/share
 
     ln -s ${cctools}/libexec $out/libexec
+
+    mkdir -p "info/nix-support" "man/nix-support"
```

Comment:
@vcunat there's no nix-level way to do per-output propgated deps, right?

Reply:
@Ericson2314: right, I think there's no better way than this ATM.  There certainly wasn't when multi-output stdenv was done (years ago), and I haven't seen any use case up to now.  If more of them appear, we can design some kind of API.